src/rpc/common/Validators.cpp

```diff
@@ -126,4 +127,42 @@ CustomValidator LedgerIndexValidator = CustomValidator{
         return MaybeError{};
     }};
 
+CustomValidator AccountValidator = CustomValidator{
+    [](boost::json::value const& value, std::string_view key) -> MaybeError {
+        if (!value.is_string())
+        {
+            return Error{RPC::Status{
+                RPC::RippledError::rpcINVALID_PARAMS,
+                std::string(key) + "NotString"}};
+        }
+        // we are using accountFromStringStrict from RPCHelpers, after we remove
```

Comment:
It's always useful to add `TODO:` in front of comments like this one so that we can see it easier later and not forget to change this. Consider adding :)